contents/Documentation/Podfile

```diff
@@ -0,0 +1,14 @@
+target 'Documentation' do
+  platform :osx, '10.14'
+  use_frameworks!
+
+  pod "Bow", "~> 0.3.0"
```

Comment:
maybe we could delete `, "~> 0.3.0"` in order to keep it updates - the user will know how to fix the version